src/coreclr/vm/frames.cpp

```diff
@@ -1125,6 +1125,8 @@ GCFrame::GCFrame(Thread *pThread, OBJECTREF *pObjRefs, UINT numObjRefs, BOOL may
     }
     CONTRACTL_END;
 
+    m_osStackLocation = this;
```

Comment:
*this* looks like it needs an ifdef guard though

Reply:
Right, this was causing the build break